ArjunaCore/txoj/README.md

```diff
@@ -1,37 +1,30 @@
-JBoss, Home of Professional Open Source Copyright 2008, Red Hat Middleware 
```

Comment:
removed copyright?

Reply:
the copyright was left as it was just it's displayed as the comment